packages/shopify-app-remix/src/index.ts

```diff
@@ -94,16 +103,34 @@ export function shopifyApp<
         keyof Config['webhooks'] | MandatoryTopics
       >(params),
     },
-    ...(isAdminApp(appConfig.distribution)
-      ? {}
-      : {login: loginFactory(params)}),
-  } as ShopifyApp<Config>;
+    boundary: {
+      error: errorBoundary,
+      headers: headersBoundary,
+    },
```

Comment:
Why return this from the `shopify` object and not a separate export? Is this just future proofing? I guess that makes sense :+1: